feature-libs/order/core/facade/replenishment-order.service.ts

```diff
@@ -22,7 +22,8 @@ import { OrderSelectors } from '../store/selectors/index';
 @Injectable()
 export class ReplenishmentOrderService implements ReplenishmentOrderFacade {
   constructor(
-    protected store: Store<StateWithOrder | StateWithProcess<void>>,
+    protected store: Store<StateWithOrder>,
+    protected processStateStore: Store<StateWithProcess<void>>,
```

Comment:
We have to remember to document these kind of changes somehow for the future 5.0 migration docs.  Any ideas?

Reply:
I can add a TODO comment to it.

Reply:
Perhaps add the TODO next to the deprecation message.  When we delete the old code, that's when we'll need to remember to produce migration doc + schematics.